platform/darwin/src/NSExpression+MGLAdditions.h

```diff
@@ -42,6 +90,105 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, readonly) id mgl_jsonExpressionObject;
 
+/**
+ Returns a constant expression containing an `NSString`.
+ 
+ This is equivalent to call `[NSExpression expressionForConstant:]`.
```

Comment:
It may already be confusing enough that there are two ways to create a string-typed constant expression (`+[NSExpression expressionWithFormat:]` and `+[NSExpression expressionForConstantValue:]`). This third way provides a little bit of type safety, but I’m not sure that it’s worth the additional complexity.

Reply:
Initially I thought  it would be unnecessary to add this convenient method, but after making examples using each format it was more clear/natural use the constructor for each type. I'm not sure all people are familiar with which flavor of NSExpression they may need, and I don't want them to spend time trying to figure it out. I think we should keep it.